label.go

```diff
@@ -75,6 +78,11 @@ func (l *Label) heightForWidth(w int) int {
 	return len(strings.Split(wordwrap.WrapString(l.text, uint(w)), "\n"))
 }
 
+// GetText returns the text content of the label.
+func (l *Label) GetText() string {
```

Comment:
I think this should be just `Text()` to be consistent with the naming scheme, e.g., `TextEdit.Text()`.

Reply:
Sounds good to me. I'll update that.